### DISCUSSION TITLE
How to add Giscus to custom content?

### DISCUSSION THREAD

**Discussion by FuadEfendi** [hbstack/theme#39] [Q&A]
Hi Razon,

I added subfolder `content/design-patterns`, how to enable Giscus on documents?

I tried this:
`
hb:
  design-patterns:
    giscus:
...` 

And this:
`
layout: docs
...
`
Thanks,

**Comment by FuadEfendi** [comment]
I think I found an answer: `type: docs` (it works!)

Do I need also `layout: post`?

**Comment by razonyang** [comment]
Yep, just specified type as docs on front matter.

> Do I need also `layout: post`?

It's unnecessary that specify the `type` or `layout` for posts layout, since it is used by default.
